wicket/test-data/sled-sp-img-1.0.0.tar.gz

```diff
@@ -0,0 +1 @@
+This is not a real SP Image. Hell it's not even a tarball!
```

Comment:
What's the over/under on this string accidentally getting flashed to an SP at some point? :-D

In seriousness, should we maybe name this with a nonsensical component to make that less likely? `test-data/fake-update-img-1.0.0.tar.gz` or something